src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', (e) => {
+    resolve('Success');
+    document.body.appendChild(resolved);
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('error'));
+    document.body.appendChild(rejected);
```

Comment:
The variable `rejected` is used here before it is defined. Consider moving the creation and configuration of the `rejected` element before this line.